a3p-integration/proposals/z:acceptance/test-lib/rpc.js

```diff
@@ -0,0 +1,291 @@
+/** @file copied from packages/agoric-cli */
+// TODO DRY in https://github.com/Agoric/agoric-sdk/issues/9109
```

Comment:
I understand why we have to duplicate stuff from `synthetic-chain`. If we can import from `@agoric/internal`, why do we need to repeat code that's in `agoric-cli`?

I looked around and didn't find where these functions were copied from.  Are they in agoric-sdk?

Reply:
That comment was kept from the original rpc.js file, which end up being deleted in a previous PR and reintroduced on this one.
I updated the description to reflect the current implementation of rpc.js.